app/src/main/res/layout/fragment_detail.xml

```diff
@@ -1,13 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".fragments.BlankFragment">
+    tools:context=".fragments.DetailsFragment">
 
     <TextView
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:id="@+id/textviewArtistName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         android:text="@string/hello_blank_fragment" />
 
-</FrameLayout>
\ No newline at end of file
+    <ProgressBar
+        android:id="@+id/progressBar2"
```

Comment:
We can extract this progress bar into a separate file and include it in whichever layout we want. This also lets us create a generic extension on activity to show and hide the indicator.